### DISCUSSION TITLE
Help to run yarle

### DISCUSSION THREAD

**Discussion by abhyudaya633** [akosbalasko/yarle#175] [Q&A]
I removed the line for 'templateFile' as I want to keep it to default. should I leave the line as it is?
And also I used both the absolute path and relative path in 'outputDir'. but both don't work.  what could I be missing?

**Comment by akosbalasko** [comment]
Hi @abhyudaya63 !
The output dir should be defined as relative path.
Could you please provide the output log and the error?  

Thank you!

**Reply by abhyudaya633** [reply]
there is no error, nothing happens. I have the attached the video file containing the config file and process that happens. 

Thank you for helping!

https://user-images.githubusercontent.com/54078716/104129087-1a355c00-5391-11eb-9a21-1670bfe559aa.mp4

**Reply by akosbalasko** [reply]
Wow, that's strange! Could you please try it by adding a fake template, just to check if the problem is related to the missing template, or something else?
Does your enex file contain huge attachments, pdfs etc? However ai stresstested the tool with up to a 3GB enex file containing afair around 100 MB attachment each, but the devil and the details, you know :)

**Reply by abhyudaya633** [reply]
I made an empty .tmpl file and wrote the absolute path in the config file but the problem remained. 
enex file is fairly small, 34.7 MB.

Could the problem be with ubuntu 20.04 or maybe node.js?

**Reply by akosbalasko** [reply]
Could you please try with only one note containing just a simple text? 
Thank you!

**Reply by abhyudaya633** [reply]
I tried it. It didn't make a difference. The same thing happened.
I would try it on a different machine soon and would report back.

**Reply by abhyudaya633** [reply]
Program worked on other machine. There must have been some problem with mine. I just transferred the md files. 

Thank you for replying and sorry for taking your time

**Comment by akosbalasko** [comment]
You're welcome, I'm sorry for did not find the root cause, but luckily you have your md files! :)
